ibis/common/validators.py

```diff
@@ -58,6 +58,57 @@ def __repr__(self):
 validator = Curried
 
 
+class instance_of(Validator):
```

Comment:
Could we keep the original validator by combining it with a lazy one?

Something like the following should work:

```py
rlz.one_of([
    rlz.instance_of(...), 
    rlz.lazy_instance_of("shapely.<...>")
])
```

Reply:
Sure, that's a simpler idea. Will do.

Reply:
done.